operator/Makefile

```diff
@@ -52,6 +52,12 @@ install-cert-manager:
 	kubectl rollout status deployment.apps/cert-manager-cainjector -n cert-manager
 	kubectl rollout status deployment.apps/cert-manager-webhook -n cert-manager
 
+
+manifests_all: manifests manifests_v1
+
+install_all: install install_v1
```

Comment:
install all doesn't make sense to me - you either need to install pre 1.18 CRD or post 1.18 CRD